paimon-web-ui-new/src/layouts/content/components/menubar/index.tsx

```diff
@@ -20,40 +20,15 @@ import { RouterLink } from "vue-router"
 import type { NavBar } from "@/store/config/type"
 
 export default defineComponent({
+  props:{
+    menuOptions: {
+      type: Array as PropType<any>,
```

Comment:
please use the naive Menu component Type
```suggestion
      type: Array as PropType<MenuOption>,
```